src/main/java/org/gridsuite/directory/server/dto/elasticsearch/DirectoryElementInfos.java

```diff
@@ -28,7 +29,7 @@
 @ToString
 @EqualsAndHashCode
 @Schema(description = "Directory element infos")
-@Document(indexName = "#{@environment.getProperty('powsybl-ws.elasticsearch.index.prefix')}directory-elements")
+@Document(indexName = "#{@environment.getProperty('powsybl-ws.elasticsearch.index.prefix')}directory-elements-v2")
```

Comment:
I see with @jonenst, and we will keep for now the same name, and we alert the team and during the next deployment that we must delete the old index.